cmake/depthaiDependencies.cmake

```
@@ -76,8 +76,9 @@ endif()
 
 # OpenCV 4 - (optional, quiet always)
 find_package(OpenCV 4 QUIET CONFIG)
-
-find_package(jsoncpp QUIET)
+if(NOT TARGET JsonCpp::JsonCpp)
+    find_package(jsoncpp QUIET)
```

@Serafadam I tried using this on OSX, but the problem persist. However I found that asking hunter to install the package does the job. So maybe this could help?

```suggestion
    hunter_add_package(jsoncpp)
    find_package(jsoncpp QUIET)
```

---

@Serafadam would you be able to check if the change from @danzvara works for you?

Using hunter to find it should be fine IMO. CC: @zrezke for v2.26